lbry/schema/mime_types.py

```diff
@@ -166,10 +168,41 @@
     '.wmv': ('video/x-ms-wmv', 'video')
 }
 
+# maps detected extensions to the possible analogs
+# i.e. .cbz file is actually a .zip
+synonyms_map = {
+    '.zip': ['.cbz'],
+    '.rar': ['.cbr'],
+    '.ar': ['.a']
+}
+
+log = logging.getLogger(__name__)
+
 
 def guess_media_type(path):
     _, ext = os.path.splitext(path)
     extension = ext.strip().lower()
```

Comment:
Since the `extension` is needed without the dot in the old code and in your code, it would make sense to slice it here to remove the dot. This will simplify your code (no need for `f".{kind.extension}"`) and the old code as well.

Reply:
the dotted extension is being used in the types_map, thats why I prepending the dot to the detected extension when setting the real_extension var, am I getting something wrong here ?

Reply:
does `synonyms_map` need to have the dots?